config/recommended.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   rules: {
     'conditional-flows': 'error',
     'end-event-required': 'error',
+    'event-scopes': 'error',
```

Comment:
```suggestion
    'event-scopes': 'warn',
```

I'd make this a warning at the core. It can happen fairly easily as a temporary issue during modeling.